### DISCUSSION TITLE
Review of ATO Tool Page in Console (Office hours: April 22, 2021)

### DISCUSSION THREAD

**Discussion by a-christensen** [twilio-labs/paste#1369] [Office hours] [answered]
**The specific questions I have are...**

- How do ya'll feel about my usage of the disclosures for the steps (since they can be done out of order)?
- Thoughts on the typography and layouts for the disclosures?
- Any other general feedback ya'll have for this page

**Background:** This page will exist to help good customers, who have been ATOed and suspended, secure their accounts and get reactivated asap. Right now this process is extremely manual and confusing (emails back and forth requesting different actions). This page seeks to make the process clear and easy to self-serve.
**Stage of design:** Exploration (pre-usability testing)
**Research (Listen stage):** https://docs.google.com/presentation/d/1pj4mKUHbV9nD56-GipXXy7vbiVznAqN9I-YyCkLKmaA/edit?usp=sharing
**Links to mockups or prototype:** https://www.figma.com/file/XvhqsNDlFd2zgldkfkctQm/ATO-Tool-Console-Suspension?node-id=44%3A12041

**Comment by gloriliale** [comment, marked as answer]
Hi @a-christensen! Thanks for posting this so that we can discuss the details in office hours.

**Reply by richbachman** [reply]
Here's the [video recording link](https://www.loom.com/share/0d363a2e1bc342c0b616da44cea7160c). Thanks for sharing. Let us know what you decide on for the illustration.

**Reply by a-christensen** [reply]
Totally! Thanks for all your feedback. I was thinking of going with this illustration for the "in review" with Fraud Ops stage. What do ya'll think?
![Newspaper-illustration](https://user-images.githubusercontent.com/52463764/116148197-d7786500-a69d-11eb-8fc3-63600bb16caa.png)
